stix_shifter_modules/secretserver/README.md

```diff
@@ -0,0 +1,95 @@
+# Secret Server
```

Comment:
Nice to have a public api documentation link or the product resource link in the readme.